compiler/lib/specialize_js.ml

```diff
@@ -23,47 +23,49 @@ open Code
 open Flow
 
 let specialize_instr info i =
-  match i with
-  | Let (x, Prim (Extern "caml_format_int", [ y; z ])) -> (
+  match i, Config.target () with
+  | Let (x, Prim (Extern "caml_format_int", [ y; z ])), `JavaScript -> (
```

Comment:
Why is this disabled ?

Reply:
@OlivierNicole, can you also add a comment explaining why it's disabled with wasm ?

Reply:
Done